scripts/Packages/Package_Gullwing__QFP_SOIC_SO/ipc_gullwing_generator.py

```diff
@@ -34,8 +34,14 @@ def __init__(self, configuration):
                 self.ipc_defintions = yaml.safe_load(ipc_stream)
 
                 self.configuration['min_ep_to_pad_clearance'] = 0.2
+
+                #ToDo: find a settings file that can contain these.
```

Comment:
Why not pile these settings into https://github.com/pointhi/kicad-footprint-generator/blob/master/scripts/Packages/ipc_definitions.yaml? It also has a `ipc_generic_rules` section. Also in `ipc_noLead_generator.py`.

Reply:
Make an issue about that as this change should really not include anything other than the changes to the ep rounding

Reply:
https://github.com/pointhi/kicad-footprint-generator/issues/435